src/script/components/accentColorPicker.ts

```diff
@@ -48,7 +48,7 @@ class AccentColorPicker {
 }
 
 // Knockout registration of the accent color picker component.
-ko.components.register('accent-color-picker', {
+ko.components.register('BBBaccent-color-picker', {
```

Comment:
What's `BBB` means?

Reply:
It's just excluded, will be removed when the PR is finished